src/app/collection/config/tab/attribute-list/attribute-list.component.html

```diff
@@ -70,14 +71,20 @@ <h6 class="dropdown-header">Lumeer suggests:</h6>
       <td class="text-right">{{ formatNumber(attribute.usageCount) }}</td>
       <td>
         <div class="d-flex flex-wrap position-relative">
-          <span class="constraint px-2 ml-1 rounded"
-                *ngFor="let constraint of attribute.constraints; let idx = index"
-                [style.background-color]="darken(constraintColor(constraint), mouseOver.attribute === attribute && mouseOver.constraint === constraint ? 40 : 0)"
-                (mouseenter)="mouseOver = {attribute: attribute, constraint: constraint}"
-                (mouseleave)="mouseOver = {attribute: null, constraint: ''}"
-                (click)="removeConstraint(attribute, idx)"
-                title="Remove Constraint"
-                i18n="@@collection.config.tab.attributes.constraint.remove">{{ constraint }}</span>
+          <div class="constraint px-2 ml-1 rounded"
+               *ngFor="let constraint of attribute.constraints; let idx = index; trackBy:trackByIndex"
```

Comment:
Because the `input` inside it modifies one of the array elements (the string `constraint`). So this way it ignores those changes and only minds the array indexes.